include/deal.II/base/cuda.h

```diff
@@ -151,6 +151,34 @@ namespace Utilities
                                                cudaMemcpyHostToDevice);
       AssertCuda(cuda_error_code);
     }
+
+    /**
+     * Copy @p size elements in @p pointer_dev to the host pointer @p pointer_host.
+     */
+    template <typename T>
+    inline void
+    copy_to_host(const T *pointer_dev, unsigned int size, T *pointer_host)
```

Comment:
I find the sequence of the arguments a bit odd. Why not do it like in the `cudaMemcpy` or `std::memcpy` case?

Reply:
Because in the library, we always do input first and output last.